code/modules/supermatter/supermatter.dm

```diff
@@ -289,7 +282,30 @@
 
 	broadcast_status()
 
-	//Ok, get the air from the turf
+	if(!istype(L)) 	// If we're in the large reinforced container, slowly delaminate if above a certain power threshold. Otherwise, weird case so we can just return.
+		if(istype(L, /obj/structure/closet/crate/secure/large) && damage > explosion_point / 10)
+			damage += max((power-1600)/10, 0)
+			power = min(power, 1600)
```

Comment:
Having a hot shard go from 0 power to 1600 explicitly because you put it in a crate/space tile seems weird to me.

On the other hand, I can already see the delamination meme engines, with mass drivers and shards cycling through cold chambers, hot chambers and space tiles for free power (half an emitter's worth at it's peak). 
Might be just what Boxer needed to finally learn an engine

Reply:
That's not what this is doing. This bit already exists in the code; it essentially adds 0.1 damage for every power over 1600, such that if it were 1654 on a tick it would reduce it to min(1650, 1600) = 1600 and then increase damage by 54/10.

Reply:
You're right, I misread that min for a max